index.html

```diff
@@ -44,15 +44,9 @@ <h4>Core Design and Fuel Analysis</h4>
 <li><a href="{{ site.url }}users/grechanp/">Pavel Grechanuk</a> (<a href="https://github.com/grechasneak" class="user-mention">@grechasneak</a>)</li>
 <li><a href="{{ site.url }}users/schicklr/">Robert Schickler</a> (<a href="https://github.com/robertschickler" class="user-mention">@robertschickler</a>)</li>
 <li><a href="{{ site.url }}users/stewryan/">Ryan Stewart</a> (<a href="https://github.com/ryanstwrt" class="user-mention">@ryanstwrt</a>)</li>
-<li><a href="{{ site.url }}users/tamashia/">Aaron Tamashiro</a> (<a href="https://github.com/aarontama" class="user-mention">@aarontama</a>)</li>
 <li><a href="{{ site.url }}users/hillszach/">Zachary Hills</a> (<a href="https://github.com/zachthills" class="user-mention">@zachthills</a>)</li>
 
 
-</ul>
```

Comment:
@rtrp/admins We should've kept the `</ul>`. Check the current page to see what happened. I'll open an issue.